### DISCUSSION TITLE
JavaScript Primer Sponsors

### DISCUSSION THREAD

**Discussion by azu** [asciidwango/js-primer#1728] [General]
このDiscussionsは、JavaScript Primer Sponsorsに関する質問、提案、特典に関するやりとりなどをする場所として公開しています。

JavaScript Primerでは、Open Collectiveを使ってSponsorを募集しています。
次のCollectiveから、jsprimerプロジェクトを毎年や毎月ごとに一定の金額を支援することができます。

- [JavaScript Primer - Open Collective](https://opencollective.com/jsprimer)

Open CollectiveはGitHub Sponsorsと似たサービスですが、特定の個人ではなくjsprimerというプロジェクトに対して支援できます。そして、Open Collectiveは支援された金額を、Contributorに対して分配することができます。
このやりとりをオープンに透明性を持ってできるのが、Open Collectiveの特徴です。

また、企業向けのスポンサーの特典として、いくつかの特典を用意しています。
詳細は[JavaScript Primerスポンサー · JavaScript Primer #jsprimer](https://jsprimer.net/intro/sponsors/)を参照してください。

**Comment by januswel** [comment]
2025/10/13 に支援した being-ish です。スポンサーとして必要な情報を次に記載しますね。
よろしくお願いいたします。

> 企業名（指定がない場合は、Open Collectiveのユーザー名）

株式会社 being-ish

> URL（指定がない場合は、Open CollectiveのプロフィールにあるウェブサイトURL）

https://being-i.sh

> ロゴ画像（指定がない場合は、Open Collectiveのプロフィール画像）

<img width="256" height="256" alt="logo-icon-primary" src="https://github.com/user-attachments/assets/37463ebf-879a-45f6-961a-892e4cdc2eb8" />

> 掲載を希望するJavaScript PrimerのページURLを1つ

https://jsprimer.net/basic/async/

「JavaScript Promise の本」時代からわかりやすく学習させていただいたので、後継？となるページを推したいです！

**Reply by azu** [reply]
サポートありがとうございます！

早速反映させていただきました！
https://jsprimer.net/
https://jsprimer.net/basic/async/

**Comment by januswel** [comment]
反映ありがとうございます！
今後も利用させていただきます 👍🏼
